homework.md

```diff
@@ -4,84 +4,84 @@
 
 What is 2 + 2?
 
-5
+4
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+White
 
 ## Question 3
 
 What is the  2 * 3 * 6 * 1 * 10 * 34?
 
-1322
+12240
 
 ## Question 4 
 
 What does HTML stand for?
 
-Hyper Tall Massive Language
-
+HyperText markup language 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
-
+It's used to style html files
 ## Question 6
 
 Using CSS, how would I select every `<h1>` tag on the page?
 
 ```css
-#h1 {
+h1 {
 
 }
-```
+
 
 ## Question 7
 
 What is JavaScript?
 
-A programming language used on the internet
+JavaScript is the programming language of the Web.
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+Helps us to share information.
+Enables programmer  to make their own changes to that information, and share it back.
+Allows us to track how information has changed between each checkpoint.
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+You make copy of it in your own repository 
 
 ## Question 10 
 
 What happens when you clone a repostory?
 
-It send it to a friend
+It pulls down a full copy of all the repository 
 
 ## Question 11
 
 What is a Pull Request?
 
-When you send a file over the internet
+A way to submit your contributions in a project. 
 
 ## Question 12
 
 What is the Git History log?
 
-It lets you see into the future
+It stores your commits. Each time you commit, Git adds your new commit to a chain of commits, one after the other.
```

Comment:
Right! but it also enables us to track how the files have been changed over time, by whom and when.